packages/block-library/src/quote/edit.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import {
+	BlockControls,
+	AlignmentToolbar,
+	RichText,
+} from '@wordpress/editor';
+
+export default function edit( { attributes, setAttributes, isSelected, mergeBlocks, onReplace, className } ) {
```

Comment:
We use a name starting with upper case for the component names. In other places we use something which would translate to `QuoteEdit` here.